proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -379,12 +379,14 @@ message Pool {
 
 // Infraction indicates the infraction a validator commited.
 enum Infraction {
+  option (gogoproto.goproto_enum_prefix) = false;
+
   // UNSPECIFIED defines an empty infraction.
-  INFRACTION_UNSPECIFIED = 0;
+  INFRACTION_TYPE_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "Infraction_INFRACTION_UNSPECIFIED"];
```

Comment:
Please do not go go proto annotations here. This was part of the reason we removed type in the first place. I think Ics should move to become inline with that's already there instead of vice versa

Reply:
Thank you for the feedback.

I understand you concern and point of view. The change in marshalling has been addressed on ICS side.